libraries/common/global-theme.js

```diff
@@ -6,6 +6,11 @@ export default function () {
     const favicon = document.getElementById("favicon");
     if (favicon) favicon.href = chrome.runtime.getURL("/images/icon-blue.png");
   }
+  const now = Date.now() / 1000;
+  // TODO: update these timestamps for year 2024
+  if (/*now < 1648911600 && now > 1648738800 && */ ["popup"].includes(location.pathname.split("/")[2])) {
+    document.documentElement.style.setProperty("--brand-orange", "#3232fd");
```

Comment:
This is not the same blue as the prerelease versions

Reply:
> This is not the same blue as the prerelease versions

Sorry if this is a dumb question that may be a bit off-topic, but, why did you say that in a review on your own PR while you could've just put that in a comment?